python-hanadb_exporter.spec

```diff
@@ -21,7 +21,7 @@
 %endif
 
 %{?!python_module:%define python_module() python-%{**} python3-%{**}}
-Name:           python-hanadb_exporter
```

Comment:
I think you forgot to change the spec file name itself, it should be `hanadb_exporter.spec` and not `python-hanadb_exporter.spec`

Reply:
True... Let me commit that